src/common/types/sqsMessage.ts

```
@@ -68,15 +70,32 @@ export const sqsPayloadSchemas = {
         url: z.string().min(1).url()
       }).optional()
     })
+  ),
+  [AvailableSQSFunctions.CreateOrgGithubTeam]: createSQSSchema(
+    AvailableSQSFunctions.CreateOrgGithubTeam, z.object({
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)